__tests__/api.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest")
+const app = require("../api")
+const db = require("../db/connection")
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET/api/topics', () => {
+    it("Should respond with an array of topic objects containing the following properties -slug -description ", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
```

Comment:
great destructuring here, lets check the length of the array before the forEach here :)